docs/tutorial/force_calibration.rst

```diff
@@ -1,380 +1,375 @@
+Next, unpack this dictionary using the unpacking operator `**`::
 
-    driving_data = f["Nanostage Position"]["Nanostage X"][start:stop]
+    >>> fit = lk.calibrate_force(**shared_parameters, active_calibration=True, distance_to_surface=distance_to_surface)
+    >>> print(fit["kappa"].value)
+    0.11662183772410809
 
-We also need to provide the sample rate at which the data was acquired, and a rough guess for the driving frequency.
-`pylake` will find an accurate estimate of the driving frequency based on this initial estimate (provided that it is close enough)::
+And compare this to the passive calibration result::
 
-    active_model = lk.ActiveCalibrationModel(driving_data.data, volts.data, driving_data.sample_rate, bead_diameter, driving_frequency_guess=37)
+    >>> fit = lk.calibrate_force(**shared_parameters, active_calibration=False, distance_to_surface=distance_to_surface)
+    >>> print(fit["kappa"].value)
+    0.11763849764570819
 
-To check the determined frequency, we can look at the determined driving frequency::
+These values are quite close.
+However, if we do not provide the height above the surface, we can see that the passive calibration result suffers much more than the active calibration result (as passive calibration fully relies on a drag coefficient calculated from the physical input parameters)::
 
-    >>> active_model.driving_frequency
-    36.95
+    >>> print(lk.calibrate_force(**shared_parameters, active_calibration=False)["kappa"].value)
+    >>> print(lk.calibrate_force(**shared_parameters, active_calibration=True)["kappa"].value)
+    0.08616565751377737
+    0.11662183772410809
 
-We can now use this model to fit the power spectrum::
+.. note::
 
-    calibration = lk.fit_power_spectrum(power_spectrum, active_model)
-    calibration
+    When hydrodynamic correction is enabled, the height parameter impacts both passive and active calibration.
+    The reason for this is that when using the more accurate theory, the expected shape of the power spectrum depends on the distance to the surface.
+    For more information on this see the :doc:`theory section on force calibration</theory/force_calibration/force_calibration>` section.
 
-And that's all there is to it.
+Fast Sensors
+------------
 
-Analogously, we can specify `hydrodynamically_correct=True` if we wish to use the hydrodynamically correct theory here.
-This fits the thermal part with the hydrodynamically correct power spectrum and also uses a hydrodynamically correct model for the peak:
+By default, the models used always incorporate a filtering effect.
+This filtering effect occurs because a fraction of the light is not measured instantaneously by most detectors.
+Fast detectors have the ability to respond much faster to incoming light resulting in no visible filtering effect in the frequency range we are fitting.
+This means that for a fast detector, we do not need to include a filtering effect in our model.
```

Comment:
If you go with my suggestion for adding an intro section, we don't need the explanation of the filtering effect here.
```suggestion
Fast detectors have the ability to respond much faster to incoming light resulting in no visible filtering effect in the frequency range we are fitting.
This means that for a fast detector, we do not need to include a filtering effect in our model.
```
We might also want to explicitly state that this depends on the particular hardware in the C-Trap (it's implied here, but let's make it super clear).